### PR TITLE
Teams image clear

### PR DESCRIPTION
**Pull request by dominiquemb**

### CODE REVIEW

File: codalab/apps/teams/forms.py [hunk 1]
```diff
@@ -1,14 +1,48 @@
...
+from django.utils.html import escape, conditional_escape
+from django.utils.encoding import force_unicode
 from .models import Team, TeamMembership
 from tinymce.widgets import TinyMCE
+from django.utils.html import format_html
+from django.utils.encoding import force_text
+from django.utils.safestring import mark_safe
 import os
 
+class CustomImageField(ClearableFileInput):
```
Comment (ckcollab) [inline codalab/apps/teams/forms.py]: This looks pretty awesome! Could someone submit a file named `<script>whatever()</script>` or something similar -- should we be worried about `mark_safe`? I don't think you can put the `/` in the filename so may not be a big deal..

File: codalab/apps/teams/forms.py [hunk 2]
```diff
@@ -1,14 +1,48 @@
...
+                substitutions['clear_template'] = clear_template % substitutions
+
+        return mark_safe(template % substitutions)
 
 class TeamEditForm(forms.ModelForm):
     name = forms.CharField(max_length=64, required=False)
     description = forms.Textarea()
     allow_requests = forms.BooleanField(required=False)
-    image = forms.ImageField(required=False)
+#    image = forms.ImageField(required=False)
```
Comment (ckcollab) [inline codalab/apps/teams/forms.py]: remove old stuff?

File: codalab/apps/teams/forms.py [hunk 3]
```diff
@@ -1,14 +1,48 @@
...
+    image = forms.ImageField(required=False,widget=CustomImageField)
```
Comment (ckcollab) [inline codalab/apps/teams/forms.py]: nitpick: space after comma :)

File: codalab/static/teams/teams.js [hunk 4]
```diff
@@ -0,0 +1,9 @@
+$(document).ready(function() {
+	$('.remove-image .btn').click(function() {
```
Comment (ckcollab) [inline codalab/static/teams/teams.js]: Very nice stuff here, thanks! Final thing, could you fix indentation here to 4 space?

File: codalab/apps/teams/forms.py (hunk 1 above)
Comment (dominiquemb) [inline codalab/apps/teams/forms.py]: I tried uploading a JS file and got this error: "Upload a valid image. The file you uploaded was either not an image or a corrupted image."

So it seems fine.

Comment (ckcollab) [inline codalab/apps/teams/forms.py]: Hmm, I mean within the file name.  Like can I name a file in such a way to inject a script tag? Probably not. 

Also just noticed `conditional_escape` being used... this code is awesome it looks like, thanks!

**Pull request merged**